### ISSUE TITLE
Clarify about learning rate.

### ISSUE THREAD

**Issue by ArvinZhuang** [castorini/docTTTTTquery#47]
Hi 
I have noticed that in the paper the learning rate is 10^-4.
However, in the training script is "constant_0_001.gin".

Just want to make sure which one can reproduce the model.

Thank you!

**Comment by rodrigonogueira4** [comment]
Sorry, there is a typo in the paper. The correct learning rate is 10^-3

Thanks for noticing this!

**Comment by ArvinZhuang** [comment]
Right, also I notice this is the learning rate used in the T5 original paper, thanks for clarifying!
